src/lang/create.ts

```diff
@@ -152,7 +155,8 @@ export function createCallExpressionStdLibKw(
   name: string,
   unlabeled: CallExpressionKw['unlabeled'],
   args: CallExpressionKw['arguments'],
-  nonCodeMeta?: NonCodeMeta
+  nonCodeMeta?: NonCodeMeta,
+  path?: Node<Identifier>[]
 ): Node<CallExpressionKw> {
```

Comment:
Figured we didn't really need a dedicated util function for that, and could just make the path and option arg. Maybe `modulePath` would be a better name?